packages/admin-frontend/pages/index.js

```diff
@@ -17,6 +17,10 @@ class IndexPage extends React.Component<{}> {
         <Link prefetch href="/admin/crop">
           <a>Crop images</a>
         </Link>
+        <br />
```

Comment:
Don't use `<br />` elements. Wrap the links in a `div` or apply some CSS instead